tests/vars_ci.yml

```diff
@@ -0,0 +1,5 @@
+streisand_ci: yes
+
+# Answer questions in playbooks/lets-encrypt.yml
```

Comment:
I think it would be easier to check for the `streisand_noninteractive` var that we already use to skip prompting during CI/automated runs.

Reply:
@cpu That's the tricky part. In Ansible, `vars_prompt` has never supported `when:` statement. So we cannot skip a prompt based on the value of a variable. The only way to prevent Ansible from asking questions is to define the variable in advance.

I found this out the hard way:

```
$ cat a.yaml 
- hosts: localhost
  connection: local
  gather_facts: no

- include: b.yaml
  when: bbb is not defined

$ cat b.yaml 
- hosts: localhost
  connection: local
  gather_facts: no

  vars_prompt:
    - name: "var_1"
      prompt: "111"

  tasks:
    - debug:
        var: var_1

$ ansible-playbook a.yaml -e bbb=defined

PLAY [localhost] ***************************************************************
111: 

```

Reply:
A related post: https://stackoverflow.com/a/41429660/1324308

Reply:
We might put the LE interview question (or questions) in the `streisand` script. It sounds like it’s hard to do the user-friendly thing if it’s in a role.

Reply:
What do you think @cpu? Are there some pitfalls I need to take care of if I move the questions from an Ansible role to shell input?

Reply:
@nopdotcom The user experience of answering questions with `vars_prompt` is fine. The problem is that the CI system cannot answer questions interactively. So we have to answer these questions in advance, if we are running in a CI environment.

Reply:
> In Ansible, vars_prompt has never supported when: statement.

Ahh! That's super annoying/surprising!!! I understand now. Thanks

> We might put the LE interview question (or questions) in the streisand script. It sounds like it’s hard to do the user-friendly thing if it’s in a role.

@nopdotcom I'm somewhat wary of doing too much in the wrapper script if it can be avoided. I think we probably need to figure out some guidelines for what should go where. Without care we'll end up reinventing much of Ansible in very brittle shell script :-) This might be one case where it makes sense but for now to keep things simple I think we should try the Ansible prompt approach.